flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/TestFlinkUpsert.java

```diff
@@ -127,8 +127,8 @@ public void testUpsertAndQuery() {
     LocalDate dt20220302 = LocalDate.of(2022, 3, 2);
 
     sql(
-        "CREATE TABLE %s(id INT NOT NULL, province STRING NOT NULL, dt DATE, PRIMARY KEY(id,province) NOT ENFORCED) "
-            + "PARTITIONED BY (province) WITH %s",
+        "CREATE TABLE %s(id INT NOT NULL, province STRING NOT NULL, dt DATE, PRIMARY KEY(id,dt) NOT ENFORCED) "
```

Comment:
thanks for making the test more readable.

Can we also change `province` to sth like `name`? Then we can use names like `Jane`, `Joe`. might be slightly easier to follow than `a`, `b`.

Reply:
Thank you for your suggestion. I have re-submitted the code to make column name and value more understandable.